docs/docs/user/hermes-mock.md

```diff
@@ -0,0 +1,105 @@
+# Hermes Mock
+
+A thin library designed to mocking Hermes in tests.
+
+## Overview
+
+Mocking is provided by `HermesMock` class, which consists of 3 parts:
+- `HermesMockDefine` 
+- `HermesMockExpect`
+- `HermesMockQuery`
+
+##### HermesMockDefine
+
+Is responsible for defining new topics on Hermes, consists of those methods:
```

Comment:
`consists of those methods` -> `provides the following methods` or `allows the following operations`